Engineering/Entities in Domain Driven Design.md

```diff
@@ -0,0 +1,47 @@
+---
+tags: engineering, Domain-Driven Design
+author: M.Vu Cuong(Jim)
+---
+
+# Entities in Domain-Driven Design
```

Comment:
We don't need the H1 as the heading will be automatically taken from the file name.